operator/v1/0000_10_config-operator_01_config.crd.yaml

```diff
@@ -1,139 +1,144 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: configs.operator.openshift.io
 spec:
   scope: Cluster
-  preserveUnknownFields: false
   group: operator.openshift.io
-  version: v1
   names:
     kind: Config
     plural: configs
     singular: config
     categories:
     - coreoperators
-  subresources:
-    status: {}
-  validation:
-    openAPIV3Schema:
-      description: Config provides information to configure the config operator.
-      type: object
-      required:
-      - spec
-      properties:
-        apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-          type: string
-        kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-          type: string
-        metadata:
-          type: object
-        spec:
-          description: spec is the specification of the desired behavior of the Config
-            Operator.
-          type: object
-          properties:
-            logLevel:
-              description: logLevel is an intent based logging for an overall component.  It
```

Comment:
was the description at wrong level before or did that change for v1?

Reply:
maybe tab vs. spaces? The left (old) version looks wrong.

Reply:
maybe, it looks badly indented only with `?w=1`